.github/workflows/release.yaml

```diff
@@ -0,0 +1,146 @@
+            EOF
+        - name: Clone Litani
+          run: sudo git clone https://github.com/awslabs/aws-build-accumulator.git /usr/libexec/litani
+        - name: Create Symlink
+          run: sudo ln -s /usr/libexec/litani/litani litani-${{ env.LITANI_VERSION }}/usr/bin/
+        - name: Copy to usr/libexec
+          run: |
+            cp -r /usr/libexec/litani/litani litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/lib litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/bin litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/templates litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/doc litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+        - name: Move man pages and docs to usr/share
+          run: |
+            ./litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/configure && ninja
+            mv litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/out/man/*.1 litani-${{ env.LITANI_VERSION }}/usr/share/man/man1
+            mv litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/out/man/*.5 litani-${{ env.LITANI_VERSION }}/usr/share/man/man5
+            mv litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/out/man/*.7 litani-${{ env.LITANI_VERSION }}/usr/share/man/man7
+        - name: Remove unnecessary files
+          run: |
+            rm -rf litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc
+            sudo rm -rf /usr/libexec/litani
+        - name: Create .deb package
+          id: create_packages
+          run: |
+            sudo dpkg-deb --build --root-owner-group litani-${{ env.LITANI_VERSION }}
+            deb_package_name="$(ls *.deb)"
+            echo "::set-output name=deb_package::$deb_package_name"
+            echo "::set-output name=deb_package_name::$deb_package_name"
+        - name: Setup deb package
+          run: |
+            sudo apt-get update
+            sudo apt install -y ./litani-${{ env.LITANI_VERSION }}.deb
+        - name: Test deb package
+          run: |
+            litani -h
+            man litani
+            litani init --project-name nyx --stages codegen verify check
+            litani add-job --command 'which brew' --pipeline-name 'siesta' --ci-stage codegen --phony-outputs phony_1
+            litani add-job --command 'which python3' --pipeline-name 'siesta' --ci-stage verify --inputs phony_1 --phony-outputs phony_2
+            litani add-job --command 'which docker' --pipeline-name 'siesta' --ci-stage check --inputs phony_2  --phony-outputs phony_3
```

Comment:
Is there a possibility `docker` might ever be missing?

Reply:
Same with brew.

I would prefer much more trivial commands like `/usr/bin/true`. And probably there only needs to be one job.

Reply:
Resolved